src/components/NewsletterSignupForm/images/newsletterLogo.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="46" height="46" viewBox="0 0 46 46">
+  <g fill="none" fill-rule="evenodd">
```

Comment:
fillRule?

Reply:
it's just SVG we import, I don't think we need to change something there